frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -21,7 +21,7 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
     return (
       <div
         className={classNames('co-status-card__health-item', className)}
-        data-test={`${title}-health-item`}
+        data-test={`${title.toLowerCase().replace(/\s/g, '-')}-health-item`}
```

Comment:
```suggestion
        data-test="health-item"
```

Reply:
@SanjalKatiyar Vineet suggested in associated testing PR that instead of having `Foo Bar-health-item`, it made more sense to have `foo-bar-health-item`.

Reply:
okay... thanks !